extension-manifest-v3/src/store/metrics.js

```diff
@@ -0,0 +1,41 @@
+import { store } from 'hybrids';
+import { Metrics as Telemetry } from '@ghostery/libs';
+
+const VELOCITIES = [
+  'active_daily_velocity',
+  'engaged_daily_velocity',
+  'engaged_daily_count',
+];
+
+const Metrics = {
+  ...VELOCITIES.reduce((all, current) => ({ ...all, [current]: [0] }), {}),
+  [store.connect]: {
+    async get() {
+      const { metrics = {} } = await chrome.storage.local.get(['metrics']);
+      VELOCITIES.forEach((velocity) => {
+        if (
+          (metrics[velocity] &&
+            metrics[velocity].length === 1 &&
+            metrics[velocity][0] === 0) ||
+          !metrics[velocity]
```

Comment:
@smalluban what's the proper way to declare an array of numbers? `[0]` does that job but initializes the array with a leading `0`, which has to be removed in here.

Reply:
Unfortunately, It is an architectural limitation. Eighter the store can use values from the definition as defaults or not. It is more common, that you want to do the first. A good example is in the documentation [here](https://hybrids.js.org/#/store/model?id=primitives-or-nested-objects-internal).